src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Taxon.xml

```diff
@@ -31,5 +31,9 @@
             <group>admin:taxon:read</group>
             <group>shop:taxon:read</group>
         </attribute>
+        <attribute name="position">
+            <group>admin:taxon:read</group>
+            <group>admin:taxon:update</group>
+        </attribute>
```

Comment:
Not related to the PR goal.

Reply:
The goal is I used on the behat test such as `@When I move down: taxon name taxon`

Reply:
Oh, I see now, thanks for the clarification :+1: